.cascade-code/_cascade/formats/modular/widgets/Featured_News_Events_Left_Col

```diff
@@ -1,97 +1,89 @@
-#macro(outputFeaturedNewsEvents $element)
-## placeholder html for left column news/event. Feeds and options for both left and primary pulled in thru different format
-##
-    <!-- formats/modular/widgets/Featured News Events Left Col -->
-    <div class="news-events-widget news-events-widget--left newsEvents">
-        <ul class="newsEventsNav">
-            <li class="active">Featured</li>
-            <li>News</li>
-            <li>Events</li>
-        </ul>
-        <ul class="newsEventsContent">
-            <a class="no-js-link" href="https://blogs.chapman.edu/happenings/" title="View All News Stories">View All News »</a>
-            <a class="no-js-link" href="https://events.chapman.edu" title="View All Event Listings">View All Events »</a>
-            <li class="active">
-                <div class="featured">
-                    <img class="image" height="130" src="" width="225" alt="Chapman University"/>
-                    <div class="date">
-                        <div class="day"></div>
-                        <div class="month"></div>
-                        <div class="year"></div>
-                    </div>
-                    <div class="title"></div>
-                    <div class="description">View the published page to see this section.</div>
-                    <div class="readMore"></div>
-                </div>
-            </li>
-            <li>
-                <div class="news clearfix">
-                    <img class="loading" src="/_files/level/img/ajax-loader.gif" alt="Loading news list..."/>
-                    <div class="story story1" itemscope="itemscope" itemtype="http://schema.org/Article">
+#*
```

Comment:
Shouldn't these two velocity formats have the .vtl extension?

Reply:
New ones should. These are updates to existing one.